package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon app.js",
+    "dev": "nodemon app.js",
```

Comment:
Heroku uses npm start always to initialize the app. so, nodemon is only used for development, so, I create a separate script "dev" to be used locally.